Source/WebCore/testing/Internals.cpp

```diff
@@ -2228,6 +2229,23 @@ auto Internals::lastAutoFillButtonType(const HTMLInputElement& element) -> AutoF
     return toInternalsAutoFillButtonType(element.lastAutoFillButtonType());
 }
 
+Vector<String> Internals::recentSearches(const HTMLInputElement& element)
+{
+    if (!element.isSearchField())
+        return { };
+
+    auto* renderer = element.renderer();
```

Comment:
Typically we have to do a “layout update” before accessing renderers. There should be lots of examples of that in this file.

Reply:
Got it; will look.